src/main.ts

```diff
@@ -51,6 +61,14 @@ async function main(): Promise<void> {
     provide: {store},
     render: (h) => h(Vue.extend(App)),
   }).$mount('#app')
+
+  if (window?.Cypress !== undefined) {
+    console.debug('Cypress detected, exposing app on window')
+    // This will allow us to test vuex store in Cypress
+    window.store = store
```

Comment:
I'm not a fan of global objects but I couldn't find a better approach 👍

Reply:
Same here, I look for alternative but couldn't find a working one. 
That's why I put it under the condition `window?.Cypress !== undefined` to not set that global var on prod